app/0.14.x/getting-started/quickstart.md

```diff
@@ -9,7 +9,7 @@ title: 5-minute Quickstart
   <a href="https://konghq.com/install/">installed Kong</a> &mdash; It should only take a minute!
 </div>
 
-In this section, you'll learn how to manage your Kong instance. First we'll
+In this section, you'll learn how to manage your Kong instance. First, we'll
```

Comment:
This same slight error occurs in prior versions of this doc (like https://github.com/Kong/docs.konghq.com/blob/master/app/0.13.x/getting-started/quickstart.md ) - please make this correction in all versions of the docs.